docs/rst/listeners/domain_listener.rst

```diff
@@ -1,4 +1,51 @@
+.. include:: ../exports/alias.include
+
 .. _listeners_domain_listener:
 
 DomainListener
 ==============
+
+|DomainListener-api| is an abstract class defining the callbacks
+that will be triggered in response to changes in the DDS network
+(discovery of domain, participants, topics, data readers, data writers,
+and arrival of new statistics data).
+By default, all these callbacks are empty and do nothing.
+The user should implement a specialization of this class overriding the callbacks
+that are needed on the application.
+Callbacks that are not overridden will maintain their empty implementation.
+
+DomainListener defines the following callbacks:
+
+* |DomainListener::on_data_available-api|:
```

Comment:
All this callbacks are printing only `DomainListener` instead of the corresponding callback. The link correctly sends to the callback.